config/vufind/searches.ini

```diff
@@ -112,6 +112,23 @@ retain_filters_by_default = true
 ; Whether the "versions" (FRBR) link and record tab are enabled. Default is true.
 ;display_versions = true
 
+; Default Solr parameters.
+;
+; Set sow=false in search to enable support for multiterm synonyms, but note the
+; caveats described at
+; https://opensourceconnections.com/blog/2018/02/20/edismax-and-multiterm-synonyms-oddities/
+;
+; Parameters can be defined for the following search contexts:
+; *                 All contexts
+; search            Search
+; retrieve          Retrieve records
+; similar           Find similar records
+; terms             Return terms
+; alphabeticBrowse  Return information from the browse index
+; workExpressions   Return work expressions (record versions)
+;default_parameters[*] = "echoParams=none&debugQuery=false"
```

Comment:
I like the idea of passing the parameters as a query string, but maybe it's worth commenting about the format in the notes above. Have you done any testing to see how multi-valued parameters need to work? Does `parse_str` work if you repeat the name, or do you need to add brackets? (I'm suspecting the latter, which is inconsistent with how Solr works and might cause confusion if we don't comment on it... though maybe there's never a need for repeating default parameters).

Reply:
Right, multivalued params would have required the bracket syntax. I decided to just parse manually to avoid needing it. I can't think of a real case where parameters would need to be repeated, but it's still a good idea to support that.

Reply:
Makes sense!